src-tauri/src/node.rs

```diff
@@ -181,7 +186,9 @@ fn create_configuration<CS: ChainSpec + 'static>(
             // TODO: Change to constrained eventually (need DSN for this)
             state_pruning: Some(PruningMode::keep_blocks(1024)),
             keep_blocks: KeepBlocks::Some(1024),
-            wasm_method: WasmExecutionMethod::Compiled,
+            wasm_method: WasmExecutionMethod::Compiled {
+                instantiation_strategy: WasmtimeInstantiationStrategy::PoolingCopyOnWrite,
+            },
             wasm_runtime_overrides: None,
             #[cfg(not(target_os = "windows"))]
```

Comment:
This time Windows should be the same as other platforms, performance issue was fixed